site/write_ups/2023/29-03-23_writeup.md

```diff
@@ -0,0 +1,127 @@
+# Data Ethics Club: [The Tech We Won't Build](https://2022.internethealthreport.org/episodes/the-tech-we-wont-build/)
+<!--Please don't edit the info panel below-->
+
+<!--
+
+```{admonition} What's this? 
+This is summary of Wednesday 28th July's Data Ethics Club discussion, where we spoke and wrote about the New Republic article [The Rise of Private Spies](https://newrepublic.com/article/161913/we-are-bellingcat-spooked-private-investigators) by Charlie Savage.
+The summary was written by Jessica Woodgate and Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+```
+
+-->
+
+
+
+
+## Discussion
+
+### Discussion
+
+## Discussion
+
+### Discussion
+
+##### Q1 What mechanisms or options exist to you in your context (work or otherwise) to resist the development of technologies you morally disagree with?
```

Comment:
```suggestion
### What mechanisms or options exist to you in your context (work or otherwise) to resist the development of technologies you morally disagree with?
```